medusa/providers/torrent/html/tvroad.py

```diff
@@ -0,0 +1,189 @@
+
+                search_params['recherche'] = re.sub(r'[()]', '', search_string)
+                response = self.session.get(self.urls['search'], params=search_params)
+                if not response or not response.text:
+                    log.debug('No data returned from provider')
+                    continue
+
+                results += self.parse(response.text, mode)
+
+        return results
+
+    def parse(self, data, mode):
+        """
+        Parse search results for items.
+
+        :param data: The raw response from a search
+        :param mode: The current mode used to search, e.g. RSS
+
+        :return: A list of items found
+        """
+        # Units
+        units = ['O', 'KO', 'MO', 'GO', 'TO', 'PO']
+
+        items = []
+
+        with BS4Parser(data, 'html5lib') as html:
+            torrent_table = html.find(class_='sortable')
+            torrent_rows = torrent_table('tr') if torrent_table else []
+
+            # Continue only if at least one release is found
+            if len(torrent_rows) < 2:
+                log.debug('Data returned from provider does not contain any torrents')
+                return items
+
+            # Catégorie, Release, Date, DL, Size, C, S, L
+            labels = [label.get_text(strip=True) for label in torrent_rows[0]('th')]
+
+            # Skip column headers
+            for row in torrent_rows[1:]:
+                cells = row('td')
+                if len(cells) < len(labels):
+                    continue
+
+                try:
+                    title = cells[labels.index('Nom')].get_text(strip=True)
+                    download = cells[labels.index('DL')].find('a')['href']
+                    download_url = download
+                    print(download_url)
+                    if not all([title, download_url]):
+                        continue
+
+                    seeders = try_int(cells[labels.index('S')].get_text(strip=True))
+                    leechers = try_int(cells[labels.index('L')].get_text(strip=True))
+
+                    # Filter unseeded torrent
+                    if seeders < self.minseed:
+                        if mode != 'RSS':
+                            log.debug("Discarding torrent because it doesn't meet the"
+                                      ' minimum seeders: {0}. Seeders: {1}',
+                                      title, seeders)
+                        continue
+
+                    size_index = labels.index('Size') if 'Size' in labels else labels.index('Taille')
+                    torrent_size = cells[size_index].get_text()
+                    size = convert_size(torrent_size, units=units) or -1
+
+                    item = {
+                        'title': title,
+                        'link': download_url,
+                        'size': size,
+                        'seeders': seeders,
+                        'leechers': leechers,
+                        'pubdate': None,
```

Comment:
The provider doesn't have pub dates? 😢

Reply:
Not actually I hope it will soon. I've requested it to devs.